code-samples/grant/python/update_grants/README.md

```diff
@@ -0,0 +1,99 @@
+# 🛡️ Update User Grant
+
+Update user grants using the Nylas Python SDK and Poetry.
+
+## 📜 Introduction
+
+The `update_user_grant` project is a Python script that leverages the Nylas Python SDK to update user grants in a Nylas account. It provides a simple way to modify and manage user grants, enabling authorized access to resources.
+
+## ✨ Features
+
+- Update user grants with various configuration options.
+- Handle Nylas API errors and edge cases.
+- Integrated with Poetry for dependency management.
+
+## ⚙️ Requirements
+
+Before using this project, ensure you have the following prerequisites:
+
+- Python 3.10 or higher
+- Make
+- Nylas API credentials (`'API_KEY'`)
+- A grant id
+
+## 🔧 Using Make
+
+You can use the `make` command for common development tasks. Here are some useful targets:
+
+- `make venv`: Create a virtual environment.
+- `make install`: Generate a `.env` file and install required dependencies.
+- `make run`: Run the script using Poetry.
+- `make clean`: Remove build, test, coverage, and Python artifacts.
+
+## 🚀 Installation
+
+1. Clone this repository to your local machine:
+
+    ```bash
+    git clone https://github.com/nylas-samples/nylas-hacktoberfest-2023.git
+    cd nylas-hacktoberfest-2023/code-samples/grant/python/update_grants
+    ```
+
+1. Set up a virtual environment and activate it:
+
+    ```bash
+    make venv
+    ```
+
+1. Install project dependencies using Poetry:
+
+    ```bash
+    make install
+    ```
+
+1. Configure your Nylas API credentials in the `.env` file by setting your `'API_KEY'` with your actual credentials.
+
+## 🏁 Getting Started
+
+After completing the installation steps, you're ready to get started:
+
+1. Run the script to update user grants:
+
+    ```bash
+    make run
+    ```
+
+    The script will update user grants based on your configuration.
+
+    ```bash
+    *** Running the app locally... ***
+
+
+    poetry run update_grants
+    User Grants Before Update:
+    Grant ID: 80c042c2-8c30-4037-bd41-d4e54c8220c6, Status: mahmoudddharmouchhh%40gmail.com%3Agoogle
```

Comment:
Same for these, remove any user specific data before we merge.

Reply:
Done!